Sources/SwiftParser/Parser.swift

```diff
@@ -237,6 +265,7 @@ public struct Parser {
 extension Parser {
   /// Retrieves the token following the current token without consuming it.
   func peek() -> Lexer.Lexeme {
+    lexemes.recordNextTokenInLookaheadTracker()
```

Comment:
I don’t think this call is necessary anymore if `LexemeSequence.peek` calls `recordNextTokenInLookaheadTracker`.